README.md

```diff
@@ -1,46 +1,46 @@
-# React Native Image Resizer
+# React Native Image Utils
 
 A React Native module that can create scaled versions of local images (also supports the assets library on iOS).
 
-## Setup
+## Getting started
 
-Install the package:
+`$ npm install react-native-image-utils --save`
 
-* 😻 React Native >= 0.40
-```
-npm install --save react-native-image-resizer
-react-native link react-native-image-resizer
-```
+### Mostly automatic installation
 
-> ⚠️ **Heads up, breaking change!** If you're upgrading *react-native-image-resizer* to version 1.0.0, please note that the response of `createResizedImage` changed. You must now read the image uri from property `uri` of the returned object. [Here is an example](https://github.com/bamlab/react-native-image-resizer/commit/15ea06d7651faf316b946170427efa90ea48dc4e). Easy, huh?
+`$ react-native link react-native-image-utils`
 
-## Older versions:
+### Manual installation
 
-* 👨 React Native >= 0.29.2 and < 0.40
-```
-npm install --save react-native-image-resizer@0.0.12
-react-native link react-native-image-resizer
-```
 
-* 👴 React Native >= 0.28 and < 0.29.2
-```
-npm install rnpm -g
-rnpm install react-native-image-resizer@0.0.12
-```
+#### iOS
 
-### Android
+1. In XCode, in the project navigator, right click `Libraries` ➜ `Add Files to [your project's name]`
+2. Go to `node_modules` ➜ `react-native-image-utils` and add `AEImageUtils.xcodeproj`
```

Comment:
Should be:
Go to `node_modules` ➜ `react-native-image-utils` ➜`ios` and add `AEImageUtils.xcodeproj`